src/Conductor.php

```diff
@@ -24,7 +27,41 @@ public function updatePackages($paths, PackageZipper $packageZipper)
 
         $results = array();
         foreach ($finder->in($paths) as $file) {
-            $results[] = $packageZipper->zip($file);
+            if ($this->output && OutputInterface::VERBOSITY_VERBOSE <= $this->output->getVerbosity()) {
+                $this->output->writeln('<info>Package file:</info> ' . $file);
+            }
+            $zip = $packageZipper->zip($file);
+
+            // remove composer cache for custom artifact packages
+            // Composer would pick cached ones instead of artifact folder
+            if (!empty($_SERVER['HOME']) && $this->fileSystem->exists($_SERVER['HOME'])) {
+                $packageDefinition = json_decode(file_get_contents($file));
+                $path = explode("/", $packageDefinition->name);
+
+                $baseCachePath = $_SERVER['HOME'] . '/.composer/cache/files/';
```

Comment:
Good point. I'll make a note of that. I think we could read `cache-files-dir` from composer config just in case. Should be OK for now.